{"name": "test_get_user_info", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestApi' object has no attribute 'add_users'", "trace": "api_client = <tests.api.test_api.TestApi object at 0x000002A1CA297770>\ntest_users = [{'email': 'vlad@mail.ru', 'firstName': 'Vlad', 'id': 1, 'lastName': 'Mizgirev', ...}, {'email': 'islam@mail.ru', 'firstName': 'Islam', 'id': 2, 'lastName': 'Vorokov', ...}]\n\n    def test_get_user_info(api_client, test_users):\n>       api_client.add_users(test_users)\nE       AttributeError: 'TestApi' object has no attribute 'add_users'\n\ntests\\api\\test_api.py:5: AttributeError"}, "start": 1731927136279, "stop": 1731927136279, "uuid": "a184fd4d-9f97-4144-bfa3-91fce17279ba", "historyId": "082858f4976c42f8802d68e7fb561c2e", "testCaseId": "082858f4976c42f8802d68e7fb561c2e", "fullName": "tests.api.test_api.TestApi#test_get_user_info", "labels": [{"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "DESKTOP-0OQD2GR"}, {"name": "thread", "value": "17788-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_api"}]}